pipeline/inferrer/palette_inferrer/app/main.py

```diff
@@ -38,12 +32,11 @@ async def main(query_url: str):
         raise HTTPException(status_code=404, detail=error_string)
 
     palette_result = await batch_inferrer_queue.execute(image)
-    logger.info(f"extracted palette from url: {query_url}")
+    logger.info(f"extracted color palette from url: {query_url}")
 
     return {
-        "palette": palette_result["lsh"],
+        "palette_embedding": palette_result["palette_embedding"],
         "average_color_hex": palette_result["average_color_hex"],
-        "hash_params": palette_hash_params,
```

Comment:
Realise this is still a draft but was my description in the issue clear of what this should be? You may well wish to rename it given that no `hash` is going on but I think we still need to associate these vectors with the $n$ and $\alpha$ parameters with which they were created

Reply:
Can't believe I didn't know about these $\sqrt{cool} \ \LaTeX \ e^x\rho ress\iota0n_s$

Reply:
Wasn't actually paying attention to what $\alpha$ did, that makes sense - sounds good, we can delete a load of code in the API, let's hard code $n_{bins}$

Reply:
🕺